Tools/AutoRest.Settings.targets

```diff
@@ -130,6 +130,9 @@
     <Message Text="Compiling Razor templates for project $(MSBuildProjectDirectory)" />
     <!-- Compile razor template. -->
     <Exec Command="$(RazorCommand) &quot;$(MSBuildProjectDirectory)\Templates&quot; $(RootNamespace).Templates" />
+    <Exec Command="$(RazorCommand) &quot;$(MSBuildProjectDirectory)\Templates\Rest\Client&quot; $(RootNamespace).Templates.Client" />
```

Comment:
Sounds good, I'll let this be here for now.

---
In reply to: [94625465](https://github.com/Azure/autorest/pull/1697#discussion_r94625465) [](ancestors = 94625465)